pulsar/consumer_partition.go

```diff
@@ -333,6 +335,7 @@ func newPartitionConsumer(parent Consumer, client *client, options *partitionCon
 	}
 	if pc.options.autoReceiverQueueSize {
 		pc.currentQueueSize.Store(initialReceiverQueueSize)
+		pc.client.memLimit.RegisterTrigger(receiverQueueShrinkMemThreshold, pc.shrinkReceiverQueueSize)
```

Comment:
This threshold should be managed by the ·memory_limit_controller·, and we don't need to set it when we register trigger.

Reply:
But `memory_limit_controller` is created by client.  If threshold is managed by `memory_limit_controller`, will the threshold become a magic number? 
I'm not sure if my thinking is correct.

Reply:
I think it can be passed in via the constructor(`NewMemoryLimitController`).

In the short, We don't have to set a threshold for each trigger, it should be unique in `memory_limit_controller`

Reply:
> 

I see. I will modify it.